sanitize_html/lib/src/sane_html_validator.dart

```diff
@@ -217,35 +220,53 @@ class SaneHtmlValidator implements html.NodeValidator {
     @required this.allowClassName,
   });
 
-  @override
-  bool allowsAttribute(html.Element element, String attribute, String value) {
-    if (!_allowedElements.contains(element.tagName)) {
-      return false;
-    }
-    if (_alwaysAllowedAttributes.contains(attribute)) {
-      return true;
-    }
-    // Special validators for id and class on all elements
-    if (attribute == 'id') {
-      return allowElementId(element.id);
+  String sanitize(String htmlString) {
+    final root = html_parser.parseFragment(htmlString);
+    _sanitize(root);
+    return formatHtmlNode(root);
+  }
+
+  void _sanitize(Node node) {
+    if (node is Element) {
+      final tagName = node.localName.toUpperCase();
```

Comment:
I wonder if it's possible to play tricks with `xmlns` to do things we don't want? Maybe we should filter everything that has:
https://pub.dev/documentation/html/latest/dom/Element/namespaceUri.html

I'm not sure what the test case would look like :)

Reply:
Uh, maybe we don't need to worry about this because the `xmlns` attribute is not allowed..

Reply:
As we ignore namespaces, both at processing and while outputting the formatted html, I think it is fair to say that tricks won't work...